ArkhamOverlay/Services/CardLoadService.cs

```diff
@@ -89,24 +200,150 @@ public class CardLoadService {
             worker.RunWorkerAsync();
         }
 
-        private void LoadPlayerCards() {
-            var worker = new BackgroundWorker();
-            worker.DoWork += (x, y) => {
-                foreach (var player in _appData.Game.Players) {
-                    if (!string.IsNullOrEmpty(player.DeckId)) {
-                        _loadingStatusService.ReportPlayerStatus(player.ID, Status.LoadingCards);
-                        try {
-                            _arkhamDbService.LoadPlayerCards(player);
-                            _loadingStatusService.ReportPlayerStatus(player.ID, Status.Finished);
+        public void LoadPlayerCards(Player player) {
+            if (player.Slots == null) {
+                _logger.LogWarning($"{player.ID} has no cards in deck.");
+                return;
+            }
+
+            _logger.LogMessage($"Loading cards for player {player.ID}.");
+
+            // TODO: Consider alternatives to loading all cards
+            var localCards = _localCardsService.LoadLocalCards();
+
+            player.SelectableCards.Loading = true;
+            try {
+                var cards = new List<Card>();
+                foreach (var slot in player.Slots) {
+                    ArkhamDbCard arkhamDbCard = _arkhamDbService.GetCard(slot.Key);
+                    if (arkhamDbCard != null) {
+
+                        FixArkhamDbCardImageSource(arkhamDbCard);
+
+                        // Override card image with local card if possible
+                        var localCard = localCards.FirstOrDefault(c => c.ArkhamDbId == arkhamDbCard.Code);
+                        if (localCard != null) {
+                            arkhamDbCard.ImageSrc = localCard.FilePath;
+                            if (localCard.HasBack) {
+                                arkhamDbCard.BackImageSrc = localCard.BackFilePath;
+                            }
```

Comment:
Right now we aren't doing anything with card backs for player cards, which long term is a mistake since there are cards like Preston's sig which have two sides. Anyway, this code here isn't being used (I think), though the fix is probably to use it rather than remove it.

Reply:
I left the logic in the common method even if the player cards don't use it right now.